### ISSUE TITLE
Styling of Strong Password Generateor is not Proper

### ISSUE THREAD

**Issue by amrahs02** [Git21221/JS-beginner-projects#255]
## What i am going to do 
the styling of this project is not proper, the password length box and output box 
is coming out of full width.

I want to make this better looking with CSS styling.

## Screenshot
![Screenshot from 2023-10-04 00-05-24](https://github.com/Git21221/JS-beginner-projects/assets/65103353/2b99ef9b-9b80-481d-a1ed-936df9acc84c)

**Comment by amrahs02** [comment]
Please assign this issue to me

**Comment by Git21221** [comment]
@amrahs02 someone was assigned this earlier. Can you make more interesting changes to the current design? Let me know

**Comment by amrahs02** [comment]
Yes, I can make the design more interesting. Please assign the task to me, and I'll work on enhancing it.
